File: MaximumGCD.java
import java.util.Scanner;

public class MaximumGCD {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int testCase = sc.nextInt();
        for (int i = 1; i <= testCase; i++) {
            int n = sc.nextInt();
            System.out.println(n / 2);
        }
        sc.close();
    }
}

File: maximum_gcd.js
function solveTheProblem() {
    var n = parseInt(readline());
    return parseInt(n / 2);
}
var testCase = parseInt(readline());
for (var i = 1; i <= testCase; i++) {
    var maximumGCD = solveTheProblem();
    print(maximumGCD);
}